### PR TITLE
Update README.md. Add brew to goreleaser.

### PR DESCRIPTION
**Pull request by adam2k**
Updating goreleaser and the README to allow for easier installation for astro-cli users.

Fixes https://github.com/astronomer/issues/issues/1368

### CODE REVIEW

**Comment by vishwas-astro** [comment]
@adam2k @paolaperaza do we need to update any external facing documentation in astronomer.io/docs as part of this?

**Comment by adam2k** [comment]
@vishwas-astro yes, I'm going to make that update in a different PR.  I'll link this PR onto another issue that I'm creating now and the issue that I have linked here I'll link it to the documentation PR to hopefully make this more clear.

File: .goreleaser.yml [hunk 1]
```diff
@@ -21,6 +21,17 @@ build:
...
+  - github:
+      owner: astronomer
+      name: homebrew-tap
+    folder: Formula
+    homepage:  https://astronomer.io
+    description: To build and run Airflow DAGs locally via docker-compose and deploy DAGs to Astronomer-managed Airflow clusters and interact with the Astronomer API.
+    test: |
+      system "#{bin}/astro version"
+    dependencies:
+    - name: go
```
Comment (andriisoldatenko) [inline .goreleaser.yml:34]: let me test... i don't think we need golang as deps, cause your brew formula take binaries:
https://github.com/astronomer/homebrew-tap/blob/master/Formula/astro.rb#L8-L10

**Comment by adam2k** [comment]
@vishwas-astro here is the documentation PR. https://github.com/astronomer/docs/pull/199

**Comment by andriisoldatenko** [comment]
wdyt to add new section instead of removing Latest/New?
image

File: README.md [hunk 2]
```diff
@@ -35,16 +35,27 @@ The Astronomer CLI is an open-source project. Installing it to your machine allo
 
 > Note: If you are an Astronomer customer, your CLI version _must_ match the version of Astronomer you are running. If you are using Astronomer Cloud, the latest version should always be correct. If you have a custom Astronomer Enterprise installation, you may be on a different version, which means you may need to ensure that your CLI and platform match up; you can check which version of Astronomer you're running by clicking the user icon in the top right corner of our UI.
 
-### Latest Version
+### Mac OSX
```
Comment (andriisoldatenko) [inline README.md:39]: @paolaperaza can you please review this document too.

**Comment by adam2k** [comment]
@andriisoldatenko I made updates to the README and moved the via curl and via homebrew sections to under the `latest` and `previous version` sections.

**Pull request merged**